### ISSUE TITLE
Allow `turbo-rails` to be excluded from new applications with the `--ignore-action-cable` flag?

### ISSUE THREAD

**Issue by dbelling** [rails/rails#51768]
### Steps to reproduce

```ruby
$ rails g new app
$ bundle install
$ rails turbo:install
$ rails c
$ irb(main):005> Turbo::StreamsChannel.signed_stream_name('foo')
(irb):5:in `<main>': uninitialized constant ActionCable (NameError)

class Turbo::StreamsChannel < ActionCable::Channel::Base
                                                  ^^^^^^
```

### Expected behavior
I'm not sure if this is a bit of a yak shave, but - in light of the intrinsic dependency that the [Hotwire Turbo ecosystem](https://turbo.hotwired.dev/) has on `ActionCable` and the current [workaround required](https://github.com/hotwired/turbo-rails/pull/601), does it make to control the inclusion/exclusion of these libraries with this flag as well?

### Actual behavior
I was attempting to reproduce the behavior reported in [thoughtbot/administrate#2548](https://github.com/thoughtbot/administrate/issues/2548), and realized that `ActionCable` is a [soft dependency](https://github.com/hotwired/turbo-rails/blob/e376852bfb273f69f4ebb54cf516b99fcbaa7acb/package.json#L17) for the `turbo-rails` library. The Gemfile generator (at present) [does not recognize](https://github.com/rails/rails/blob/dcfabdbe4e7e53fc757d0ebf89c77349899a35bb/railties/lib/rails/generators/app_base.rb#L497) the `--ignore-action-cable` flag when adding `turbo-rails` and `stimulus-rails` to the Gemfile.

### System configuration
**Rails version**: `7.1.3.2`

**Ruby version**: `3.2.2`

**Comment by skipkayhil** [comment]
If I understand correctly, a new release of `turbo-rails` would resolve this right? I'm not sure there's anything to change here for Rails itself

**Comment by jeromedalbert** [comment]
Now that Rails 8 includes Turbo by default, this issue is becoming more apparent. I submitted a similar issue https://github.com/rails/rails/issues/51860 where the Rails app cannot boot in production if action cable is skipped. I don't know what the best solution would be, but I have mentioned @dbelling's suggestion as one of them.

**Comment by skipkayhil** [comment]
This should be fixed in `turbo-rails` `2.0.6`
